code-challenges.test.js

```diff
@@ -21,30 +20,120 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+    // PSUEDO CODE
+    // create function that takes an array
+    // removes first item from array
+    // shuffles array
+    // return new array
+    // for the test, expect statement should be flexible on order of returened array in expected output
+
+    describe('rm_First_Element_Shuffle_Array', () => {
+        it('takes an array, removes the first element and shuffles the remaining contents', () => {
+            expect(rm_First_Element_Shuffle_Array(colors1)).not.toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]));
```

Comment:
When using the `not` jest matcher, you will need to only type the value that you are not expecting to show in the array. Did your test pass after creating the function?

Reply:
yes, the test passed